mappings/net/minecraft/client/InteractionManager.mapping

```diff
@@ -1,17 +1,60 @@
 CLASS net/minecraft/class_504 net/minecraft/client/InteractionManager
 	FIELD field_2104 minecraft Lnet/minecraft/client/Minecraft;
+	FIELD field_2105 noTick Z
 	METHOD <init> (Lnet/minecraft/client/Minecraft;)V
 		ARG 1 minecraft
+	METHOD method_1705 resetBlockBreaking ()V
```

Comment:
[cancelBlockBreaking](https://maven.fabricmc.net/docs/yarn-1.21.1+build.3/net/minecraft/client/network/ClientPlayerInteractionManager.html#cancelBlockBreaking()) in Yarn